src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,56 @@
+package calculator;
+
+public class Calculator {
+
+    public static final String DELIMITER = ",|:";
+
+    public static int cal(String text) {
+        String customDelimiter = "";
+
+        if (isBlank(text)) {
+            return 0;
+        }
+        if (text.startsWith("//")){
+            String[] result = split(text, "//|\n");
+            customDelimiter = result[1];
+            text = result[2];
+
+            return sum(toInts(split(text, customDelimiter)));
+        }
+
+        return sum(toInts(split(text, DELIMITER)));
+    }
+
+    private static String[] split(String text, String delimiter) {
+        return text.split(delimiter);
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static int sum(int[] numbers) {
+        int sum = 0;
+
+        for(int number : numbers){
+            sum += number;
+        }
+        return sum;
+    }
+
+    private static int[] toInts(String[] values){
+        int[] numbers = new int[values.length];
+        for(int i=0; i<values.length; i++){
+            numbers[i] = Integer.parseInt(values[i]);
+            checkNegative(numbers[i]);
+        }
+
+        return numbers;
+    }
+
+    private static void checkNegative(int number) {
+        if(number < 0){
+            throw new RuntimeException();
+        }
+    }
```

Comment:
```suggestion
    private static void checkNegative(int number) {
        if(number < 0){
            throw new RuntimeException();
        }
    }
```
`PositiveNumber`가 있다면 해당 객체의 책임을 맡아서 진행할수 있을것 같아요!
양수만 가지는 객체를 만들어보는거죠!